builder/FindGlobals.cmake

```diff
@@ -116,20 +115,20 @@ if( Linux OR Darwin )
     append("--sysroot=${CMAKE_FIND_ROOT_PATH} " LINK_FLAGS)
   endif (DEFINED CMAKE_FIND_ROOT_PATH)
 
-  if(__ARCH MATCHES ia32)
-    append("-m32 -g" CMAKE_C_FLAGS)
-    append("-m32 -g" CMAKE_CXX_FLAGS)
-    append("-m32 -g" LINK_FLAGS)
-  else ()
+  if(CMAKE_SYSTEM_PROCESSOR MATCHES x86_64)
     append("-m64 -g" CMAKE_C_FLAGS)
     append("-m64 -g" CMAKE_CXX_FLAGS)
     append("-m64 -g" LINK_FLAGS)
+  else ()
+    append("-m32 -g" CMAKE_C_FLAGS)
+    append("-m32 -g" CMAKE_CXX_FLAGS)
+    append("-m32 -g" LINK_FLAGS)
   endif()
 
-  if(__ARCH MATCHES ia32)
-    link_directories(/usr/lib)
-  else()
+  if(CMAKE_SYSTEM_PROCESSOR MATCHES x86_64)
     link_directories(/usr/lib64)
```

Comment:
Ubuntu use another one-library dir. Can we make differentiation between CentOS and Ubuntu here?

Reply:
Is this statement not what '$(LIBDIR)' stands for, running CMAKE ?

Reply:
This may conflict with @uartie change introduced by [pr19](https://github.com/Intel-Media-SDK/MediaSDK/pull/19)

Reply:
is there any need to precise the system libraries ?

Reply:
The current change focuses on removing __ARCH. So, we are having here a side discussion. Which is good, but will be addressed in a separate change. I also don't understand this place, it needs a review and rewrite. I will create and issue to continue discussion.

Reply:
Let's track this in #126.